### ISSUE TITLE
hypercerts-sdk interface and mock implementation

### ISSUE THREAD

**Issue by ryscheng** [Network-Goods/hypercerts#53]
For front-end testing purposes, we need a mock.
This will also help us modify/adjust the frontend pilot application in parallel to the contract/sdk development

**Comment by ryscheng** [comment]
Closing this in favor of
https://github.com/Network-Goods/hypercerts-sdk/issues/4
https://github.com/Network-Goods/hypercerts-sdk/issues/3
https://github.com/Network-Goods/hypercerts-sdk/issues/1
